### PR TITLE
EPMRPP-64345 - make decision modal: increase ga coverage

### PR DESCRIPTION
**Pull request by chivekrodis**

### CODE REVIEW

File: app/src/pages/inside/stepPage/stepGrid/defectType/autoAnalyzedLabel/autoAnalyzedLabel.scss [hunk 1]
```diff
@@ -17,7 +17,7 @@
 .aa-label {
   display: inline-block;
   height: 20px;
-  min-width: 43px;
+  width: 43px;
```
Comment (AmsterGet) [inline app/src/pages/inside/stepPage/stepGrid/defectType/autoAnalyzedLabel/autoAnalyzedLabel.scss:20]: Why are you reverting your previous changes?

File: app/src/pages/inside/stepPage/modals/makeDecisionModal/makeDecisionModal.jsx [hunk 2]
```diff
@@ -261,12 +263,33 @@ const MakeDecision = ({ data }) => {
...
+    return isEqual(itemData.issue, modalState.source.issue) && issueActionType
+      ? onApplyAndContinue(defectFromTIGroup, hasSuggestions, issueActionLabel, defectFromTIGroup)
+      : onApply(
+          section,
+          defectFromTIGroup,
+          hasSuggestions,
+          optionLabel,
+          selectedItemsLength,
+          timestamp,
+        );
```
Comment (AmsterGet) [inline app/src/pages/inside/stepPage/modals/makeDecisionModal/makeDecisionModal.jsx:292]: Please move the condition upper and use `if() {} else {}` structure:
```javascript
...
if (isEqual(itemData.issue, modalState.source.issue) && issueActionType) {
const issueActionLabel = issueActionType && actionMessages[issueActionType].defaultMessage;
  eventInfo = onApplyAndContinue(defectFromTIGroup, hasSuggestions, issueActionLabel, defectFromTIGroup);
} else {
   ...
   eventInfo = onApply(
          section,
          defectFromTIGroup,
          hasSuggestions,
          optionLabel,
          selectedItemsLength,
          timestamp,
        );
}
```

**Review by AmsterGet — CHANGES_REQUESTED** [review]

File: app/src/pages/inside/stepPage/modals/makeDecisionModal/optionsSection/itemsList/itemsListHeader/itemsListHeader.jsx [hunk 3]
```diff
@@ -51,6 +58,20 @@ export const ItemsListHeader = ({
     setItems({
       selectedItems: isAllSelected ? allSelectedItems : testItems,
     });
+    // TODO optionsValue && ... remove error for bulk operation, remove after bulk ga has added
```
[the remark this answers is not in the document]
Comment (chivekrodis) [reply app/src/pages/inside/stepPage/modals/makeDecisionModal/optionsSection/itemsList/itemsListHeader/itemsListHeader.jsx:61]: construction `optionValue && ` is removing errors in console for bulk operation during clicking on the checkbox. But after adding event for bulk modal it should be refactor it

File: app/src/pages/inside/stepPage/modals/makeDecisionModal/optionsSection/optionsBlock/optionsBlock.jsx [hunk 4]
```diff
@@ -148,6 +149,8 @@ export const OptionsBlock = ({ optionValue, currentTestItem, loading, setModalSt
           testItems: [],
           selectedItems: [],
         });
+    eventsInfo.onDecisionOption &&
+      trackEvent(eventsInfo.onDecisionOption(defectFromTIGroup, messages[value].defaultMessage));
```
[the remark this answers is not in the document]
Comment (chivekrodis) [reply app/src/pages/inside/stepPage/modals/makeDecisionModal/optionsSection/optionsBlock/optionsBlock.jsx:153]: I guess no, bcs all options should have unique `ownValue`. And we have no case when no one option is selected

File: app/src/pages/inside/stepPage/stepGrid/defectType/autoAnalyzedLabel/autoAnalyzedLabel.scss (hunk 1 above)
Comment (chivekrodis) [reply app/src/pages/inside/stepPage/stepGrid/defectType/autoAnalyzedLabel/autoAnalyzedLabel.scss:20]: It looks ugly in firefox browser

**Review by AmsterGet — APPROVED** [review]

**Pull request merged**